Idoit.API.Client/CMDB/ObjectTypes/Response/Result.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Newtonsoft.Json;
+
+namespace Idoit.API.Client.CMDB.ObjectTypes.Response
+{
+    public class Result
+    {
+        [JsonProperty("id")]
+        public int id { get; set; }
+        [JsonProperty("title")]
+        public string title { get; set; }
+        [JsonProperty("container")]
+        public string container { get; set; }
+        [JsonProperty("Const")]
+        public string Const { get; set; }
```

Comment:
Why is this the only property with a big letter in the beginning?

Reply:
Hello @yhildebrandt, because const is a modifier keyword in C#, do you think it would be better if I change the variable to _const?

Reply:
@OKT90 Hm I dont know. This doesn't matter I think. You can leave it as you want 😄

Reply:
Does "Const" stand for "Constant"? Could you just say "constant" then?